specification/cognitiveservices/data-plane/Language/preview/2021-07-15-preview/analyzeconversations.json

```diff
@@ -146,27 +152,32 @@
         }
       }
     },
-    "LUISV2Parameters": {
-      "description": "This is a set of request parameters for LUIS Generally Available projects and API version v2.0.",
-      "x-ms-discriminator-value": "luis_v2",
+    "LUISParameters": {
+      "description": "This is a set of request parameters for LUIS Generally Available projects.",
+      "x-ms-discriminator-value": "luis",
+      "type": "object",
       "allOf": [
         {
           "$ref": "#/definitions/AnalyzeParameters"
         }
       ],
       "properties": {
-        "projectParameters": {
-          "type": "object",
-          "$ref": "#/definitions/LUISV2ProjectParameters"
+        "query": {
```

Comment:
Is this required? Please author all `required` parameters in both request and response models.